blockers.yaml.example

```diff
@@ -42,6 +44,15 @@ job5:
         - name: Experimental Job
         - url: <URL>
 
+# You can have build number for blockers included in report
+job5:
+    bz:
+        - 0
+    jira:
+        - 'RHOSINFRA-753'
+    builds:
+        - 2,3
+
```

Comment:
There's now two "job5"s - just make a new example at the bottom of the file please